endoreg_db/models/metadata/sensitive_meta_logic.py

```
@@ -1,31 +1,121 @@
+    from .sensitive_meta import SensitiveMeta  # Import model for type hinting
 
 logger = logging.getLogger(__name__)
 SECRET_SALT = os.getenv("DJANGO_SALT", "default_salt")
 DEFAULT_UNKNOWN_NAME = "unknown"
 
+# Regex-Pattern für verschiedene Datumsformate
+ISO_RX = re.compile(r"^\d{4}-\d{2}-\d{2}$")
+DE_RX = re.compile(r"^\d{2}\.\d{2}\.\d{4}$")
+
+
+def parse_any_date(s: str) -> Optional[date]:
```

Pylint (4.0.2)

[refactor] Too many return statements (8/6)

(R0911)